client/src/components/admin/tables/ProjectRow.tsx

```diff
@@ -4,16 +4,23 @@ import DeletePopup from './DeletePopup';
 import EditProjectPopup from './EditProjectPopup';
 import useAdminStore from '../../../store';
 import { postRequest } from '../../../api';
+import { getRequest } from '../../../api';
+import FlagsPopup from '../FlagsPopup';
 
 interface ProjectRowProps {
     project: Project;
     idx: number;
+    flags: Flag[];
     checked: boolean;
     handleCheckedChange: (e: React.ChangeEvent<HTMLInputElement>, idx: number) => void;
 }
 
-const ProjectRow = ({ project, idx, checked, handleCheckedChange }: ProjectRowProps) => {
+const ProjectRow = ({ project, idx, flags, checked, handleCheckedChange }: ProjectRowProps) => {
     const [popup, setPopup] = useState(false);
+    const [showFlags, setShowFlags] = useState(false);
```

Comment:
Can we keep this state variable name consistent with the `editPopup` and `deletePopup` naming -- rename to `flagPopup`.